toolchest_client/api/query.py

```diff
@@ -229,7 +238,11 @@ def _send_initial_request(self, tool_name, tool_version, tool_args, database_nam
         try:
             create_response.raise_for_status()
         except HTTPError:
-            logger.error("Query creation failed.", file=sys.stderr)
+            error_message = "."
```

Comment:
I see how `"."` fits as the default value here, but reading this is a bit confusing – I'd opt for keeping this as `""` and not having a final period if there's a 4xx error